### PR TITLE
Disambiguate plurals

### PR DESCRIPTION
**Pull request by daankoning**
Fixes (part of ) #2234

### CODE REVIEW

File: tabbycat/results/templates/public_enter_results.html [hunk 1]
```diff
@@ -61,7 +61,8 @@
         <input id="id_debate_result_status" type="hidden" name="debate_result_status" value="{{ debate.STATUS_DRAFT }}" />
         <input id="id_discarded" type="hidden" name="discarded" />
         <input id="id_confirmed" type="hidden" name="confirmed" />
-        <input class="save btn btn-success btn-block " type="submit" value="{% trans 'Submit Ballot(s)' %}" tabindex="{{ form.nexttabindex }}"/>
+        {% blocktranslate count counter=form.scoresheets|length asvar button_text %} Submit Ballot {% plural %} Submit Ballots {% endblocktranslate %}
```
Comment (tienne-B) [inline tabbycat/results/templates/public_enter_results.html:64]:
```suggestion
        {% blocktranslate trimmed count counter=form.scoresheets|length asvar button_text %}
          Submit Ballot
        {% plural %}
          Submit Ballots
        {% endblocktranslate %}
```

File: tabbycat/breakqual/base.py [hunk 2]
```diff
@@ -90,11 +90,23 @@ def _metric_name(metric):
...
+                "The break qualification rule %(rule)s requires the following "
+                "metrics to be in the team standings precedence in order to "
+                "work: %(required)s;",
+                len(self.required_metrics),
+            )
+            missing_metrics_message = ngettext(
+                " and the following is missing: %(missing)s.",
+                " and the following are missing: %(missing)s.",
+                len(missing_metrics),
+            )
```
Comment (tienne-B) [inline tabbycat/breakqual/base.py:106]: I'd rather rephrase the message to only require one list than split it in two, as the ordering of clauses (or even agglutination) could determine how they're combined differently than in English. Maybe something like
> The %(rule)s break qualification rule is missing the following required metric in the team standings precedence: %(missing)s

@czlee ?

Comment (czlee) [reply tabbycat/breakqual/base.py:106]: Yeah, this kind of thing was a struggle for me in several places with internationalization. I agree, you can't assume that you can just concatenate sentences like that in other languages (though I can't actually name you a counterexample language off the top of my head for this case).

This error message evidently tried to be detailed enough to be as helpful as possible, but yeah, I agree that dropping the `required` list would be a minimal sacrifice in this context.

While I'm here, thanks so much for doing this @daankoning!!

**Comment by daankoning** [comment]
I thought I'd be able to fix it before the weekend (and went away on a tournament), evidently I was not 😅. 

As @tienne-B points out in #2234  the "Send Message(s)" button requires some JS which I do not have enough experience to figure out. Someone else would have to do that if they deem it important enough to fix.

File: tabbycat/breakqual/base.py (hunk 2 above)
Comment (daankoning) [reply tabbycat/breakqual/base.py]: Implemented this!

**Pull request merged**